### PR TITLE
Getting top tasks from AEM

### PR DESCRIPTION
**Pull request by nibivi77**
### Description

Fetching top tasks from AEM

### Additional Notes

- Added a call for the dictionary in AEM. That is where all titles and other miscellaneous things will reside. There are different ways to implement it so please be constructively judgmental.

### CODE REVIEW

File: __tests__/pages/home.test.js [hunk 1]
```diff
@@ -83,6 +83,28 @@ describe('Home page', () => {
     },
   ]
 
+  const topTasks = [
+    {
+      properties: {
```
Comment (MarcoGoC) [inline __tests__/pages/home.test.js:88]: needs to be updated , alpha has 'benefit' instead of 'properties'

File: pages/home.js [hunk 2]
```diff
@@ -13,7 +13,27 @@ import fr from '../locales/fr'
 
 export default function Home(props) {
   const t = props.locale === 'en' ? en : fr
+  const topTasks = () => {
+    return props.topTasks?.map((task) => {
+      try {
+        let title = task.properties.elements.scTitleEn.title
```
Comment (MarcoGoC) [inline pages/home.js:19]: same as with the test 'benefit' instead of properties once alpha is working

File: pages/home.js [hunk 3]
```diff
@@ -112,12 +120,38 @@ export async function getStaticProps({ locale }) {
     benefits = AEMbenefits.data.entities
   }
 
+  // Get list of top tasks
+  let topTasksReturned = await aemService.getFragment(
+    'alpha/components/top-tasks.json'
```
Comment (MarcoGoC) [inline pages/home.js:125]: base api variable has alpha in it

Comment (nibivi77) [reply pages/home.js:125]: I need to wait for your PR to be merged before I can remove the alpha or else the benefits inside of CardList.js crashes

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**